src/store/actions/load-all-words.js

```diff
@@ -0,0 +1,32 @@
+import { WordsService } from '../../service/get-all-words';
+
+export const LOAD_ALL_WORDS_REQUEST = 'LOAD_ALL_WORDS_REQUEST';
+export const LOAD_ALL_WORDS_SUCCESS = 'LOAD_ALL_WORDS_SUCCESS';
+export const LOAD_ALL_WORDS_FAILURE = 'LOAD_ALL_WORDS_FAILURE';
+
+
+const loadRequest = () => ({
+    type: LOAD_ALL_WORDS_REQUEST,
+    fetching: true,
+});
+
+const loadSuccess = (allWords) => ({
+    type: LOAD_ALL_WORDS_SUCCESS,
+    data: allWords,
+    fetching: false,
+});
+
+const loadFailure = () => ({
+    type: LOAD_ALL_WORDS_FAILURE,
+    fetching: false,
+    error: true,
+});
```

Comment:
```
loadRequest: type
loadSuccess: type + data
loadFailure: type + error (text message)
```

the above describes simplified actions (no `fetching`)
setting `fetching` to the right value should be done by reducers

I think of it this way:
- actions (type + payload) are descriptions of what happened in the app
- reducers receive actions and translate them into state changes

example: sending an action `LOAD_ALL_WORDS_REQUEST` without any payload is a complete information that a request started, it's reducer's job to interpret it and decide that `fetching` should be set to `true` now

the difference may look subtle here but this stems from general philosophy around redux

image having `RESET_WHOLE_APP` action in some app, it looks like this action does not need payload, then it would be reducer's task to decide that number of state properties should be set to default: score reducer, time reducer, user info reducer, preferences reducer etc.

Reply:
makes sense :)